podcasts/PlaylistCellView.swift

```diff
@@ -5,6 +5,19 @@ struct PlaylistCellView: View {
     @EnvironmentObject var theme: Theme
     @ObservedObject var viewModel: PlaylistCellViewModel
 
+    @State private var isOn: Bool
+    private let onToggleChange: ((Bool) -> Void)?
```

Comment:
Would it make sense to combine these into a Binding?

The callsite (`PlaylistSelectionViewController`) would then need to define a binding like so:
```
let isOn = Binding<Bool>(
    get: { [weak self] in
        guard let self = self else { return false }
        return self.selectedPlaylists.contains(playlist.uuid)
    },
    set: { newValue in
        onToggleChange(newValue)
    }
)
```

This would probably be better for further SwiftUI usage. Here's a patch:
[BindingChange.patch](https://github.com/user-attachments/files/22346567/BindingChange.patch)